src/EventStore.Core/TransactionLog/Chunks/TFChunk/TFChunk.cs

```diff
@@ -968,6 +924,7 @@ private enum CacheStatus {
 			bool closed = true;
 			closed &= TryDestructFileStreams();
 			closed &= TryDestructMemStreams();
+			_handle?.Dispose();
```

Comment:
i think we only want to dispose this if `closed` is true (otherwise we still have some active readers)

Reply:
:+1: